src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,8 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        ColorSupplier colorSupplier = new ColorSupplier();
+        System.out.println(colorSupplier.getRandomColor());
+        System.out.println(Lottery.getRandomBall().toString());
     }
```

Comment:
To test the program, in main() method, create **three** balls using getRandomBall() method from Lottery class.
(it is better to use a cycle for balls creation)
also, can we write just `System.out.println(Lottery.getRandomBall());`? (without `toString`)